### PR TITLE
doc: Add Regional limitations of RF use in LoRaWAN

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Article on different regional limitations of RF use in LoRaWAN.

#### Screenshots

image

#### Changes

- New document
- ...

#### Notes for Reviewers

@benolayinka please review.

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by pradeeka7** [comment]
ETSI EN 300 220-2 V3.2.1 (2018-06)
HARMONISED EUROPEAN STANDARD - Short Range Devices (SRD) operating in the frequency range 25 MHz to 1 000 MHz; 
Part 2: Harmonised Standard for access to radio spectrum for non specific radio equipment
[en_30022002v030201p.pdf](https://github.com/TheThingsNetwork/docs/files/10670083/en_30022002v030201p.pdf)

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+weight:
+---
+
+The sub-gigahertz ISM (Industrial, Scientific, and Medical) band is a range of radio frequencies between **902 MHz and 928 MHz** in the **United States**, and between **868 MHz and 870 MHz** in **Europe**, that is available for unlicensed use. The frequencies mentioned are commonly used for wireless communications in industrial, scientific, and medical applications, and are also used in LoRaWAN networks.
+
+# Europe (863-870 MHz)
+
+In Europe, LoRaWAN uses a range of **24 to 80 channels** with either 125 kHz or 250 kHz bandwidth, and 250 kHz bandwidth is only applied with DR (Data Rate) 6.
+
+All LoRaWAN end devices are required to operate at least on channels 868.10 MHz, 868.30 MHz, and 868.50 MHz. These channels must also have a bandwidth of 125 kHz at DR0 to DR5 and be restricted to a maximum [duty cycle](/lorawan/duty-cycle/) of 1%. Gateways must be listening on one or more of these channels. The same channels are used to send the join-request message and in that case, the duty cycle should be as follows:
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: > Gateways must be listening on one or more of these channels.

Gateway should listen on all three these channels.

Comment (jpmeijers) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: I think we are seeing it the wrong way around here. The channels that are used are dictated by which channels a gateway (and therefore network) listens on. The three default channels are defined like this so that we are sure the device's first uplinks/joins are on one of the three channels we are guaranteed that the gateway is listening on.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- Aggregated during the first hour following power-up or reset: Transmit time < 36Sec
+- Aggregated during the next 10 hours: Transmit time < 36Sec
+- After the first 11 hours, aggregated over 24h: Transmit time < 8.7Sec per 24h
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: <36s in the first hour (3600s) makes sense, as 36s/3600s = 1%. But how do we get to <36s in the next 10 hours, and 8.75s in 24h? Are we taking the TTN fair use policy into account here, or some best practice?

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The [LoRa Alliance®](https://lora-alliance.org/) recommends a duty cycle limitation of 1% in the European band which means that a device can transmit for no more than 1% of the time while ensuring the maximum EIRP (Effective Isotropic Radiated Power) of +16 dBm. The purpose of the duty cycle limitation is to ensure that devices operating in the ISM band do not cause harmful interference to other devices operating in the same band. 
+
+The EIRP of 16 dBm is roughly equivalent to 40 mW and is the same as an ERP of 14 dBm, which is approximately equivalent to 25 mW. To learn more about EIRP and ERP, you can refer to the [LoRaWAN®](/lorawan/) section of The Things Network.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: > The EIRP of 16 dBm is equal to 40 mW and is equivalent to an ERP of 14 dBm, which is equal to 25 mW. To learn more about EIRP and ERP, you can refer to the [LoRaWAN®](/lorawan/) section of The Things Network.

Conversion from dBm to mW is exact, so do not use "roughly" or "equivalent". It's a factual conversion, with some rounding. Converting from EIRP to ERP is an estimate, as we are ignoring the 0.15 of the 2.15dB gain of a dipole.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+However, the [ETSI](https://www.etsi.org/) (European Telecommunications Standards Institute) has segmented the European ISM frequency band into 6 sub-bands (K, L, M, N, P, Q) and imposes additional restrictions on the duty cycle and maximum ERP for some sub-bands (Please see pages 21-22 of the [ETSI EN300.220 V3.2.1 (2018-06)](https://www.etsi.org/deliver/etsi_en/300200_300299/30022002/03.02.01_60/en_30022002v030201p.pdf)) document.
+
+- K (863 MHz - 865 MHz): 0.1%, 25 mW ERP
+- L (865 MHz - 868 MHz): 1%, 25 mW ERP
+- M (868 MHz - 868.6 MHz): 1%, 25 mW ERP
+- N (868.7 MHz - 869.2 MHz): 0.1%, 25 mW ERP
+- P (869.4 MHz - 869.65 MHz): 10%, 500 mW ERP
+- Q (869.7 MHz - 870 MHz): 1%, 25 mW ERP
+
+For example, devices operating in the 869,4-869,65 MHz range, which is sub-band P, have a duty cycle limitation of 10%, which means that a device can transmit for no more than 10% of the time.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: Rather be consistent with the decimal separator here. Use `.` not `,`.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Usually, the limitations of sub-band K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. LoRaWAN's RX2 channel is mainly used for downlink communication because there are more end devices with individual duty cycles compared to gateways, and the RX2 channel is in a specific frequency sub-band for gateways to handle the imbalance between end devices and gateways.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: > LoRaWAN's RX2 channel is mainly used for downlink communication

Well, RX2 is by definition downlink. So maybe rephrase this. Something like: "There are normally many more end device thatn gateways on a LoRaWAN network. Gateways can therefore run into their duty cycle limit quicker than end devices. LoRaWAN's RX2 channel is normally chose to fall in sub-bad P to benefit from the higher duty cycle limit, and provide 10x more downlinks to gateways than to end devices to combat this imbalance."

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+By default, the RX1 receive window for downlinks uses the same frequency as the preceding uplink, while the RX2 receive window uses a fixed frequency, with the default being 869.525 MHz at DR 0 using SF12 with 125 kHz.
+
+The default frequency for beacon broadcasts and ping-slot transmissions for Class B end devices in Europe is 869.525 MHz. For LoRaWAN relays, channel 0 uses 865.1 MHz and 865.5 MHz for WOR (Wake On Radio) transmissions respectively, and channel 1 uses 865.3 MHz and 865.9 MHz for WOR ACK (Wake On Radio Acknowledgement) transmissions respectively.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: The word "respectively" is out of place here. We normally use it when saying something like: "A and B and used by X an Y respectively." where X uses A and Y uses B. These sentences do not have this double reference.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- There are **8 uplink channels**, numbered from 64 to 71, each with a bandwidth of 500 kHz. These channels start at a center frequency at 903.0 MHz and increment linearly by 1.6 MHz up to 914.2 MHz.
+- There are **8 downlink channels**, numbered from 0 to 7, each with a bandwidth of 500 kHz. These channels start at a center frequency at 923.3 MHz and increment linearly by 600 kHz up to 927.5 MHz.
+
+The end devices must have the capability to store parameters for the 72 uplink channels mentioned above.
+
+The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that the devices are able to successfully transmit data while avoiding interference with other devices.
+
+The value of dwell time for LoRaWAN for the US902-928 band is 400ms. This means that the device should spend 400 ms on each channel before switching to the next channel.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: > should spend 400 ms on each channel

I interpret the rules differently. When I google most people says something like
> dwell time shall not exceed 400 mS

So in other words, we are not required to stay the full 400ms on the channel. We can stay there shorter and than hop to another frequency. So using the words "should spend" is incorrect.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+Both end devices and gateways are also limited to a maximum EIRP of +30 dBm. The maximum EIRP can be maintained by adjusting the transmitter power, antenna gain, or cable loss. 
+
+The downlink channel for the RX1 receive window is based on the uplink channel that was used to send the preceding uplink. The default downlink channel for the RX2 receive window is 923.3 MHz with a data rate of DR8.
+
+For Class B beacons, the US902-928 band uses 8 frequencies 923.3, 923.9, 924.5, 925.1, 925.7, 926.3, 926.9, 927.5 starting at 923.3 MHz on a rotation basis. For LoRaWAN relays, channel 0 uses 916.7 MHz and 919.9 MHz for WOR transmissions respectively, and channel 1 uses 918.3 MHz and 921.5 MHz for WOR ACK transmissions respectively.
+
+It is also important to note that there may be additional regulatory requirements and certifications that must be met in order to legally operate in the ISM band in specific countries. It is highly recommended to have your LoRaWAN end device [certified](https://lora-alliance.org/lorawan-certification/) by the LoRa Alliance® before releasing it to the market.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: LoRa Alliance certification is not good enough. You device also needs to be FCC/ICASA/RED/R&TTE/etc approved/certified.

For a quick summary see: https://www.element.com/nucleus/2018/wireless-devices-a-guide-to-compliance-in-the-us-and-eu
and https://electronics.stackexchange.com/a/186516/225812

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md (hunk 2 above)
Comment (pradeeka7) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: See page 65 of https://lora-alliance.org/resource_hub/lorawan-specification-v1-1/
image

Comment (jpmeijers) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: That table is for "Retransmissions back-off", not duty cycle in general.

Comment (pradeeka7) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: So then shall I remove it from the document?

Comment (jpmeijers) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: Yes maybe just omit this to avoid confusion.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+The downlink channel for the RX1 receive window is based on the uplink channel that was used to send the preceding uplink. The default downlink channel for the RX2 receive window is 923.3 MHz with a data rate of DR8.
+
+For Class B beacons, the US902-928 band uses 8 frequencies 923.3, 923.9, 924.5, 925.1, 925.7, 926.3, 926.9, 927.5 starting at 923.3 MHz on a rotation basis. For LoRaWAN relays, channel 0 is used for WOR transmissions with a frequency range of 916.7 MHz to 919.9 MHz, while channel 1 is used for WOR ACK transmissions with a frequency range of 918.3 MHz to 921.5 MHz.
+
+It is important to note that there may be additional regulatory requirements and certifications that must be met to legally operate in the ISM band, such as obtaining approval or certification from organizations such as the [FCC, ICASA, RED, R&TTE, etc](https://www.element.com/nucleus/2018/wireless-devices-a-guide-to-compliance-in-the-us-and-eu). Getting your LoRaWAN end device [certified](https://lora-alliance.org/lorawan-certification/) by the LoRa Alliance® before releasing it to the market is also highly recommended.
```
Comment (jpmeijers) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: This paragraph is applicable to all regions, so maybe we should put this under a dedicated heading.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md (hunk 6 above)
Comment (jpmeijers) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: I'd rephrase this paragraph as follows:

> In Europe 869.525 MHz is used for [Class B beacon transmissions and ping slots](https://lora-developers.semtech.com/documentation/tech-papers-and-guides/lorawan-class-b-devices/).  

Reading up on the relay functionality, I can not find a reference to the WOR frequency. In which document is it? 
Also I only see a mention of one default WOR channel, with the second defined by the network. You list 4 frequencies for 2 channels. Something here doesn't add up. If there's confusion or doubt here, maybe better to leave this part about relays out for now.

https://resources.lora-alliance.org/technical-specifications/ts011-1-0-0-relay

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Different Regional Limitations of RF Use in LoRaWAN"
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
title: "Regional Limitations of RF Use in LoRaWAN"
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+section: Additional Information
+description: ""
+weight:
+---
+
+The sub-gigahertz ISM (Industrial, Scientific, and Medical) band is a range of radio frequencies between **902 MHz and 928 MHz** in the **United States**, and between **868 MHz and 870 MHz** in **Europe**, that is available for unlicensed use. The frequencies mentioned are commonly used for wireless communications in industrial, scientific, and medical applications, and are also used in LoRaWAN networks.
+
+# Europe (863-870 MHz)
+
+In Europe, LoRaWAN uses a range of **24 to 80 channels** with either 125 kHz or 250 kHz bandwidth, and 250 kHz bandwidth is only applied with DR (Data Rate) 6.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
In Europe, LoRaWAN uses a range of **24 to 80 channels** with either 125 kHz or 250 kHz bandwidth, where 250 kHz bandwidth can be achieved only using DR (Data Rate) 6.
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+All LoRaWAN end devices are required to operate at least on channels 868.10 MHz, 868.30 MHz, and 868.50 MHz. These channels must also have a bandwidth of 125 kHz at DR0 to DR5 and be restricted to a maximum [duty cycle](/lorawan/duty-cycle/) of 1%. Gateways should listen on these three channels. The same channels are used to send the join-request message and in that case, the duty cycle should be as follows:
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
All LoRaWAN end devices are required to operate at least on channels 868.10 MHz, 868.30 MHz, and 868.50 MHz. These channels must have a bandwidth of 125 kHz at DR0 to DR5, and be restricted to a maximum [duty cycle](/lorawan/duty-cycle/) of 1%. These three channels are used for sending join-requests, so all gateways should listen on these channels too.

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+
+- Aggregated during the first hour following power-up or reset: Transmit time < 36Sec
+- Aggregated during the next 10 hours: Transmit time < 36Sec
+- After the first 11 hours, aggregated over 24h: Transmit time < 8.7Sec per 24h
+
+The [LoRa Alliance®](https://lora-alliance.org/) recommends a duty cycle limitation of 1% in the European band which means that a device can transmit for no more than 1% of the time while ensuring the maximum EIRP (Effective Isotropic Radiated Power) of +16 dBm. The purpose of the duty cycle limitation is to ensure that devices operating in the ISM band do not cause harmful interference to other devices operating in the same band. 
+
+The EIRP of 16 dBm is equal to 40 mW and is equivalent to an ERP of 14 dBm, which is equal to 25 mW. To learn more about EIRP and ERP, you can refer to the [LoRaWAN®](/lorawan/) section of The Things Network.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
The EIRP of 16 dBm is equal to 40 mW and is equivalent to an ERP of 14 dBm, which is equal to 25 mW. To learn more about EIRP and ERP, you can refer to the [LoRaWAN®](/lorawan/) section.
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+
+However, the [ETSI](https://www.etsi.org/) (European Telecommunications Standards Institute) has segmented the European ISM frequency band into 6 sub-bands (K, L, M, N, P, Q) and imposes additional restrictions on the duty cycle and maximum ERP for some sub-bands (Please see pages 21-22 of the [ETSI EN300.220 V3.2.1 (2018-06)](https://www.etsi.org/deliver/etsi_en/300200_300299/30022002/03.02.01_60/en_30022002v030201p.pdf)) document.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
However, [ETSI](https://www.etsi.org/) (European Telecommunications Standards Institute) has segmented the European ISM frequency band into 6 sub-bands (K, L, M, N, P, Q) and imposes additional restrictions on the duty cycle and maximum ERP for some sub-bands (see pages 21-22 of the [ETSI EN300.220 V3.2.1 (2018-06) document](https://www.etsi.org/deliver/etsi_en/300200_300299/30022002/03.02.01_60/en_30022002v030201p.pdf)).
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+- K (863 MHz - 865 MHz): 0.1%, 25 mW ERP
+- L (865 MHz - 868 MHz): 1%, 25 mW ERP
+- M (868 MHz - 868.6 MHz): 1%, 25 mW ERP
+- N (868.7 MHz - 869.2 MHz): 0.1%, 25 mW ERP
+- P (869.4 MHz - 869.65 MHz): 10%, 500 mW ERP
+- Q (869.7 MHz - 870 MHz): 1%, 25 mW ERP
+
+For example, devices operating in the 869.4-869.65 MHz range, which is sub-band P, have a duty cycle limitation of 10%, which means that a device can transmit for no more than 10% of the time.
+
+Usually, the limitations of sub-band K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. There are normally many more end device thatn gateways on a LoRaWAN network. Gateways can therefore run into their duty cycle limit quicker than end devices. LoRaWAN's RX2 channel is normally chose to fall in sub-bad P to benefit from the higher duty cycle limit, and provide 10x more downlinks to gateways than to end devices to combat this imbalance.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
Usually, the limitations of sub-bands K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. Usually, a number of end devices in a LoRaWAN network is way higher than a number of gateways. Gateways can therefore run into their duty cycle limit quicker than end devices. LoRaWAN's RX2 channel is normally chosen to fall into sub-bad P in order to benefit from the higher duty cycle limit, and provide 10x more downlinks to gateways instead of end devices.
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+
+By default, the RX1 receive window for downlinks uses the same frequency as the preceding uplink, while the RX2 receive window uses a fixed frequency, with the default being 869.525 MHz at DR 0 using SF12 with 125 kHz.
+
+The default frequency for beacon broadcasts and ping-slot transmissions for Class B end devices in Europe is 869.525 MHz. For LoRaWAN relays, channel 0 uses 865.1 MHz and 865.5 MHz for WOR (Wake On Radio) transmissions respectively, and channel 1 uses 865.3 MHz and 865.9 MHz for WOR ACK (Wake On Radio Acknowledgement) transmissions respectively.
+
+# US (902-928 MHz)
+
+The following regulations are applied to the USA, Canada, and all other countries in ITU Region 2 adopting the entire [FCC](https://www.fcc.gov/) (Federal Communications Commission) 47 CFR Part 15 regulations in the 902-928 ISM band.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
The following regulations are applied to the USA, Canada, and all other countries in ITU Region 2 adopting the entire [FCC](https://www.fcc.gov/) (Federal Communications Commission) 47 CFR (Code of Federal Regulations) Part 15 in the 902-928 ISM band.
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+
+In the United States, the ISM band is divided into **80 channels**: 72 for uplink and 8 for downlink.
+
+- There are **64 uplink channels**, numbered from 0 to 63, each with a bandwidth of 125 kHz. These channels start at a center frequency at 902.3 MHz and increment linearly by 200 kHz up to 914.9 MHz.
+- There are **8 uplink channels**, numbered from 64 to 71, each with a bandwidth of 500 kHz. These channels start at a center frequency at 903.0 MHz and increment linearly by 1.6 MHz up to 914.2 MHz.
+- There are **8 downlink channels**, numbered from 0 to 7, each with a bandwidth of 500 kHz. These channels start at a center frequency at 923.3 MHz and increment linearly by 600 kHz up to 927.5 MHz.
+
+The end devices must have the capability to store parameters for the 72 uplink channels mentioned above.
+
+The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that the devices are able to successfully transmit data while avoiding interference with other devices.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
The dwell time for LoRaWAN in the US902-928 band refers to the amount of time that a device spends on a specific channel before switching to the next channel. The dwell time is an important parameter for LoRaWAN devices as it helps to ensure that devices are able to successfully transmit data while avoiding interference with other devices.
```

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+
+For the US ISM band, the [dwell time](https://www.thethingsindustries.com/docs/reference/glossary/) for transmitting a LoRaWAN message must not exceed 400 ms. It is important to consider this dwell time limitation when designing and operating devices. For example, if a device is transmitting data at a high rate, it may need to use a faster hopping rate in order to comply with the 400 ms dwell time limitation.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: if you want to link something on dwell time, then please do it on the first occurrence of that term

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md [hunk 19]
```diff
@@ -0,0 +1,63 @@
...
+
+Both end devices and gateways are also limited to a maximum EIRP of +30 dBm. The maximum EIRP can be maintained by adjusting the transmitter power, antenna gain, or cable loss. 
+
+The downlink channel for the RX1 receive window is based on the uplink channel that was used to send the preceding uplink. The default downlink channel for the RX2 receive window is 923.3 MHz with a data rate of DR8.
```
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md:55]: please be consistent - either use "DR8" or "DR 8", don't mix them

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md (hunk 9 above)
Comment (nejraselimovic) [inline doc/content/lorawan/regional-limitations-of-rf-use/_index.md]:
```suggestion
It is important to note that there may be additional regulatory requirements and certifications that must be met to legally operate in the ISM band, such as obtaining approval or certification from organizations such as [FCC, ICASA, RED, R&TTE, etc](https://www.element.com/nucleus/2018/wireless-devices-a-guide-to-compliance-in-the-us-and-eu). Getting your LoRaWAN end device [certified](https://lora-alliance.org/lorawan-certification/) by the LoRa Alliance® before releasing it to the market is also highly recommended.
```

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: doc/content/lorawan/regional-limitations-of-rf-use/_index.md (hunk 9 above)
Comment (pradeeka7) [reply doc/content/lorawan/regional-limitations-of-rf-use/_index.md]: @jpmeijers I removed the WOR frequencies from the document; however, they are mentioned in the [LoRaWAN® Regional Parameters RP002-1.0.4 document](https://resources.lora-alliance.org/technical-specifications/rp002-1-0-4-regional-parameters).
- for EU - page 36
- for US - page 43
